lib/esm/command/territory/upgrade.rb

```diff
@@ -24,27 +24,39 @@ class Upgrade < ApplicationCommand
         #################################
 
         def on_execute
-          deliver!(
-            function_name: "upgradeTerritory",
-            territory_id: arguments.territory_id,
-            uid: current_user.steam_uid
+          response = call_sqf_function(
+            "ESMs_command_upgrade",
+            territory_id: arguments.territory_id
           )
```

Comment:
is end parenthesis on new line for multiple arguments a personal style preference or a best practice?

Reply:
Personal style preference. This is "C-style".